Add default value to IMGPROXY_MAX_REDIRECTS documentation and clarify usage of value 0 (#969)
imgproxy/imgproxy: docs/configuration.md

Requests to some image sources may go through too many redirects or enter an infinite loop. You can limit the number of allowed redirects:

> * `IMGPROXY_MAX_REDIRECTS`: the max number of redirects imgproxy can follow while requesting the source image. When set to `0`, no redirecting is allowed. Default: `10`

Reviewer: ```suggestion
* `IMGPROXY_MAX_REDIRECTS`: the max number of redirects imgproxy can follow while requesting the source image. When set to `0`, no redirects are allowed. Default: `10`
```
